test/integration/smoke/test_scale_vm.py

```diff
@@ -599,7 +599,7 @@ def test_04_scale_vm_with_user_account(self):
             self.cleanup.append(self.bigger_offering)
         else:
             self.bigger_offering = self.big_offering
-
+        self.cleanup.append(self.virtual_machine_in_user_account)
```

Comment:
this seems out of place and the assignment and appending of `self.big_offering` also looks iffy. Note that `self.big_offering` is obfuscating `cls.big_offering` but `self.cleanup.big_offering` is not obfuscating `cls._cleanup.big_offering` (and it shouldn't)
```suggestion
```
this line should move up to just below line 551